app/components/edition/reorder_component.rb

```diff
@@ -9,6 +9,10 @@ def initialize(edition:, order:, redirect_path:)
 
   attr_reader :edition, :order, :redirect_path
 
+  def item_keys
+    order + (edition.default_order - order)
```

Comment:
would it perhaps be helpful to extract `(edition.default_order - order)` to a method with a friendly name to make it clear what's being done here and why? If I understand correctly, that might be something like `elements_added_during_reordering()`?